applications/neuroscan/frontend/package.json

```diff
@@ -9,7 +9,7 @@
     "@material-ui/lab": "^4.0.0-alpha.60",
     "@metacell/geppetto-meta-client": "0.0.8",
     "@metacell/geppetto-meta-core": "0.0.8",
-    "@metacell/geppetto-meta-ui": "0.0.8",
+    "@metacell/geppetto-meta-ui": "0.0.7-rc3.0",
```

Comment:
why are we releasing with a release candidate version? imo we should release production software with only stable versions